Tensile/SolutionStructs.py

```diff
@@ -2639,6 +2639,12 @@ def assignDerivedParameters(state):
     if state["WavefrontSize"] == 32 and state["KernelLanguage"] == "Source":
       reject(state, "WavefrontSize=32 not yet supported for source kernels.")
 
+    if state["KernelLanguage"] == "Source" \
```

Comment:
Is this a temporal check?
I think it is better to add some comment to remove this temporal check later once the issue is fixed.